test/test_autograd.py

```diff
@@ -4758,6 +4758,44 @@ def test_grad_fn_attr_bindings(self):
         with self.assertRaisesRegex(RuntimeError, "after they have already been freed"):
             out.grad_fn._saved_weight
 
+    def test_custom_function_saved_tensors(self):
+        def getFn(save_for_backward=None):
+            class MyFn(Function):
+                @staticmethod
+                def forward(ctx, x):
+                    if save_for_backward:
+                        ctx.save_for_backward(*eval(save_for_backward))
+                    return x
+
+                @staticmethod
+                def backward(ctx, g):
+                    return g, None
```

Comment:
Why is there two returns here while you have a single input for the forward?

Note to self: I still don't understand why we allow trailing None here :p